crowbar_framework/app/models/node_object.rb

```diff
@@ -479,6 +483,7 @@ def validate_public_name(value, unique_check = true)
   def update_public_name(value)
     unless value.nil?
       crowbar["crowbar"]["public_name"] = value
+      @notify_on_save["public_name"] = true
```

Comment:
It might be a stupid question: Are we sure that `update_public_name` is only called whenever the value of it has changed? I'm not seeing the same check-if-changed mechanism as earlier.